lua/entities/gmod_tardis/modules/sh_tracking.lua

```diff
@@ -0,0 +1,510 @@
+-- Tracking
+
+local TRACKING_MAX_DISTANCE_SET = 1000
```

Comment:
This should take into account the bounding box of the target so we can target very large props

Reply:
https://github.com/MattJeanes/TARDIS/pull/969/commits/4abca165a3bc2241764f4910ce0d5f9bd4afacb6